### PR TITLE
proxy transparent for context broker

### PR DESCRIPTION
**Pull request by josecarlosaparicio**

### CODE REVIEW

**Comment by mrutid** [comment]
what about use /proxy/cb, /proxy/keystone... and so on.

So /proxy allocate us in the proxy handler
/cb allows to retrieve CB config host port...

File: lib/service.js [hunk 1]
```diff
@@ -63,7 +66,33 @@ function start(config) {
...
+
+    targetContextBroker = config.client.contextBroker.protocol + '://' + 
+        config.client.contextBroker.host + 
+        (config.client.contextBroker.port ? ':' + config.client.contextBroker.port : '');
+
+    app.use('/proxy/**', proxy({
+            target: 'target', 
+            changeOrigin: true,
+            pathRewrite: {
+                '^/proxy/cb' : '',
```
Comment (mrutid) [inline lib/service.js]: Here you don't know it /cb need to be removed. you know it inside the case expresion.

**Comment by mrutid** [comment]
Refactor will be needed. But go ahead!

LGTM

**Comment by fgalan** [comment]
LGTM also to the new version of the code at this point.

File: lib/service.js (hunk 1 above)
Comment (fgalan) [reply lib/service.js]: Fixed in e9b24ca

**Pull request merged**